crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -810,7 +810,7 @@ type Owner implements ObjectOwner {
 	"""
 	coinConnection(first: Int, after: String, last: Int, before: String, type: String): CoinConnection
 	"""
-	The stake objects for the given address
+	The `0x3::staking_pool::StakedSui` objects owned by the given object.
```

Comment:
Probably should have done a separate PR, but I saw that this is one place where the comment is different than the other two existing `stakedSuiConnection` comments, so decided to update it.

Reply:
I'll let it slide ;) thanks for the clean-up!